Libs/MRML/Widgets/qMRMLThreeDView.cxx

```diff
@@ -430,20 +430,6 @@ vtkMRMLViewNode* qMRMLThreeDView::mrmlViewNode()const
   return d->MRMLViewNode;
 }
 
-// --------------------------------------------------------------------------
-void qMRMLThreeDView::lookFromViewAxis(const ctkAxesWidget::Axis& axis)
-{
-  Q_D(qMRMLThreeDView);
-  if (!d->MRMLViewNode)
-    {
-    qCritical() << "qMRMLThreeDView::lookFromViewAxis: no valid view node.";
-    return;
-    }
-  double fov = d->MRMLViewNode->GetFieldOfView();
-  Q_ASSERT(fov >= 0.0);
-  this->lookFromAxis(axis, fov);
```

Comment:
@lassoan I removed this `lookFromViewAxis` because it would just be a pass through to `lookFromAxis` because the FOV usage is deprecated.

Reply:
For backward compatibility, what about  keeping `qMRMLThreeDView::lookFromViewAxis` so that it calls `lookFromAxis(axis)`. Optionally, we could also log a deprecation message like `qWarning() << "This function is deprecated. Use lookFromAxis(const ctkAxesWidget::Axis& axis) instead";`

Reply:
Searching through existing code, the function is being used in a few extension:
* https://github.com/search?l=Python&q=lookFromViewAxis&type=Code
* https://sourcegraph.com/search?q=context:global+lookFromViewAxis+lang:python&patternType=standard

For the sake of addressing the runtime warning, for now, I would suggest to keep the function and have it call `lookFromAxis`

Reply:
@jcfr I've restored `qMRMLThreeDView::lookFromViewAxis` and have a runtime deprecation message to specify to use  `lookFromAxis` instead.